prime-dotnet-webapi/Configuration/EnrolleeProfileVersionConfiguration.cs

```diff
@@ -0,0 +1,41 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Serialization;
+using Prime.Models;
+
+namespace Prime.Configuration
+{
+    public class EnrolleeProfileVersionConfiguration : IEntityTypeConfiguration<EnrolleeProfileVersion>
+    {
+        // private DefaultContractResolver _contractResolver = new DefaultContractResolver
+        // {
+        //     NamingStrategy = new CamelCaseNamingStrategy()
+        // };
```

Comment:
commented code?

Reply:
This would be a better implementation, but couldn't get this to work and had to do it in the service.  Wanted to see if anyone had suggestions why this isn't working.